### PR TITLE
Add tuned status field

### PR DESCRIPTION
**Pull request by yanirq**
Add under the performance profile CR an additional field that would indicate the tuned CR related to it. e.g.:
#oc describe performanceprofile <profile name> | grep Tuned
Tuned:  tuned namespace/tuned name

### CODE REVIEW

File: functests/3_performance_status/status.go [hunk 1]
```diff
@@ -34,6 +37,29 @@ var _ = Describe("Status testing of performance profile", func() {
 		nodeAnnotationDesiredConfig := "machineconfiguration.openshift.io/desiredConfig"
 		nodeLabel := map[string]string{fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerCNF): ""}
 
+		It("Tuned status field tied to Performance Profile", func() {
```
Comment (gsr-shanks) [inline functests/3_performance_status/status.go:40]: Can we please add test_id here as OCP-30894

Comment (yanirq) [reply functests/3_performance_status/status.go]: @gsr-shanks  done.

File: deploy/crds/performance.openshift.io_performanceprofiles_crd.yaml [hunk 2]
```diff
@@ -166,6 +166,10 @@ spec:
                 - type
                 type: object
               type: array
+            tuned:
+              description: tuned represents the current generate tuned custom resource
```
Comment (MarSik) [inline deploy/crds/performance.openshift.io_performanceprofiles_crd.yaml:170]:
```suggestion
              description: tuned points to the Tuned custom resource object that contains the tuning values generated by this operator
```

File: functests/3_performance_status/status.go [hunk 3]
```diff
@@ -34,6 +37,29 @@ var _ = Describe("Status testing of performance profile", func() {
...
+					fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerCNF): "",
+				},
+			)
+			Expect(err).ToNot(HaveOccurred())
+			key := types.NamespacedName{
+				Name:      components.GetComponentName(testutils.PerformanceProfileName, components.ProfileNamePerformance),
+				Namespace: components.NamespaceNodeTuningOperator,
+			}
+			tuned := &tunedv1.Tuned{}
+			err = testclient.Client.Get(context.TODO(), key, tuned)
```
Comment (dshchedr) [inline functests/3_performance_status/status.go:52]: how about using testclient.GetWithRetry(...) ??

Comment (yanirq) [reply functests/3_performance_status/status.go]: added

**Pull request merged**